tests/test_user.py

```diff
@@ -117,21 +117,11 @@ async def test_get_user_while_logged_in(
 # test_x_authz_route_y is for testing the is_authorized dependency
 async def test_authz_route_no_headers(client: AsyncClient):
     res = await client.post(f"/user/{uuid4()}/follow")
-    assert res.status_code == 400
+    assert res.status_code == 422
```

Comment:
FastAPI returns 422 for invalid headers because it comes under validation error.

We can add a custom error handler if we want to return 400 instead.